app-antivirus/clamav/clamav-1.2.0.recipe

```diff
@@ -0,0 +1,128 @@
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/Cisco-Talos/clamav/archive/refs/tags/clamav-$portVersion.tar.gz"
+CHECKSUM_SHA256="efe7e234fe29a96bf4da23336e38ea2989ed0be162342993e99a11901524f43f"
+SOURCE_DIR="clamav-clamav-$portVersion"
+PATCHES="haiku-$portVersion.patch"
+
+ARCHITECTURES="all ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/clamav/clamd.conf keep-old
+	settings/clamav/freshclam.conf keep-old
+	"
+
+PROVIDES="
+	clamav$secondaryArchSuffix = $portVersion
+	cmd:clamav_config$secondaryArchSuffix = $portVersion
+	cmd:clambc$secondaryArchSuffix = $portVersion
+	cmd:clamconf$secondaryArchSuffix = $portVersion
+	cmd:clamd$secondaryArchSuffix = $portVersion
+	cmd:clamdscan$secondaryArchSuffix = $portVersion
+	cmd:clamdtop$secondaryArchSuffix = $portVersion
+	cmd:clamscan$secondaryArchSuffix = $portVersion
+	cmd:clamsubmit$secondaryArchSuffix = $portVersion
+	cmd:freshclam$secondaryArchSuffix = $portVersion
+	cmd:sigtool$secondaryArchSuffix = $portVersion
+	lib:libclamav$secondaryArchSuffix = $portVersion
+	lib:libclamunrar$secondaryArchSuffix = $portVersion
+	lib:libclamunrar_iface$secondaryArchSuffix = $portVersion
+	lib:libfreshclam$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libjson_c$secondaryArchSuffix
+	lib:libmspack$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpcre2_8$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	clamav${secondaryArchSuffix}_devel = $portVersion
+	cmd:clamav_config$secondaryArchSuffix = $portVersion
+	devel:libclamav$secondaryArchSuffix = $portVersion
+	devel:libclamunrar$secondaryArchSuffix = $portVersion
+	devel:libclamunrar_iface$secondaryArchSuffix = $portVersion
+	devel:libfreshclam$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	clamav$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcheck$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libjson_c$secondaryArchSuffix
+	devel:libmspack$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libpcre2_8$secondaryArchSuffix
+	devel:libpthread_stubs$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:python3
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. -DCMAKE_BUILD_TYPE=Release \
+		$cmakeDirArgs \
+		-DENABLE_MILTER=OFF \
+		-DENABLE_EXTERNAL_MSPACK=ON \
+		-DAPP_CONFIG_DIRECTORY=$settingsDir/clamav \
+		-DDATABASE_DIRECTORY=$settingsDir/clamav/db
```

Comment:
Need to check this later, maybe having it installed in the default path solves sed'n the config file?